Testing/VelaptorTests/Input/MouseTests.cs

```diff
@@ -148,7 +149,9 @@ public void GetState_WhenGettingMouseWheelData_ReturnsCorrectResult()
     public void PushReactable_WithOnNextMessageActionAndInvalidMouseButton_ThrowsException()
     {
         // Arrange
-        const string expected = $"The enum '{nameof(MouseButton)}' is out of range.";
+        var invalidMouseButton = 1234;
```

Comment:
Please convert to a `const`

```suggestion
        const invalidMouseButton = 1234;
```